go/flags/endtoend/vtbackup.txt

```diff
@@ -195,6 +195,7 @@ Flags:
       --remote_operation_timeout duration                           time to wait for a remote operation (default 15s)
       --restart_before_backup                                       Perform a mysqld clean/full restart after applying binlogs, but before taking the backup. Only makes sense to work around xtrabackup bugs.
       --s3_backup_aws_endpoint string                               endpoint of the S3 backend (region must be provided).
+      --s3_backup_aws_minimum_partsize int                          Minimum part size to use (default 5242880)
```

Comment:
Nit, but we use `min` in other existing flags so we can shorten this one as well. New flags should also use dashes rather than underscores as we want to transition to the latter for everything.

Reply:
Since this flag is part of a group... we may actually just want to keep them all uniform...

Reply:
I can change it to `min`, but do let me know if the preferred is to use the dashes instead of underscores for it :)

Reply:
I would favor uniformity for this config group over moving to dashes, and move these all together.

Reply:
I would probably spell out what the default is and add a period to be consistent with other flags in the same group. The default shown by the helper is not very intuitive at first.

Reply:
updated it :)